engine/engine.go

```diff
@@ -71,6 +72,9 @@ type Options struct {
 	GitClient git.Git
 	// DockerOptions is used to customize docker operations.
 	DockerOptions docker.Options
+	// TimeoutSeconds is a limit to how long a docker image pull will last
+	// If no value is provided, it defaults to 3600
+	TimeoutSeconds int
```

Comment:
Why not just make this a `time.Duration`? That's what it's meant to be might as well store it as that. Also means you don't need to convert it every time you use it

Reply:
sounds good, i used `time.Duration` all over but i needed to use `int` for the types used to parse the `.tbrc.yml`. i was getting an unmarshal error for some reason

Reply:
yeah it needs to be an int for the config, but for the Engine field it should be a time.Duration. Basically at the config layer we read it as an int but then convert it to a duration and use it as that in the rest of the app